### ISSUE TITLE
Feature Request - Tab key navigation of panel

### ISSUE THREAD

**Issue by vinsworldcom** [molsonkiko/JsonToolsNppPlugin#10]
Currently, when the panel is displayed, it seems nothing has focus.  When I launch the panel, typing does nothing anywhere.  I would expect perhaps focus could be set on the query text box so typing would immediately enter a query rather than having to mouse click on the query text box to start typing.

If I'm in the query text box, `Tab` enters an actual tab character.  Are tabs required / supported in Remes Query Language?  If spaces can be used instead, suggest that `Tab` key is used to switch to the next UI element (tabstops) as is typically the case in GUI applications.

1) Query text box
2) Submit query button
3) Query to CS button
4) Save Query res button
5) View all subtrees checkbox
6) tree view
7) Current Path button
8) current path text box
9) ... back to beginning (query text box)

Cheers.

**Comment by molsonkiko** [comment]
Hi Vince, I agree that all the issues you described are frustrating. Unfortunately, there doesn't seem to be a good way to implement *any* of those proposals. 

I have tab stops on all of the controls, I explicitly set the `AcceptsTab` parameter of the query box to `false`, which is supposed to make it so that `Tab` navigates to the next control in the tab order, and I tried adding a line that activates the query box when the tree viewer is initialized. Nothing works.

FWIW, I looked at CSVLint and CSV Query, two other C# plugins that have dockable controls, and they have the same issue you just articulated where you __can__ use tabs for all the __undocked__ forms (e.g., the settings form) but you __can't__ use tabs for any of the __docked__ forms (e.g., query box for CSV Query). It's probably just an issue with how Windows Forms integrates with Notepad++.

I'm not going to close this issue, because it may be worth revisiting in the future, but I don't think it's solvable without some pretty involved work on the underlying infrastructure.
